apps/redi-talent-pool/src/pages/app/cv-builder/cv-list/CvListPage.scss

```diff
@@ -1,3 +1,4 @@
 .heading {
   font-family: Avenir LT Std;
+  padding-top: 2rem;
```

Comment:
Please note that the `heading` class is used in two places in the `CvListPage` component. Do we want to apply this padding in both cases?